src/components/cards/FaucetMarketingCard.vue

```diff
@@ -59,15 +60,19 @@ export default {
     }
   },
   computed: {
-    ...mapState('auth', ['balance', 'registryContractApproved']),
+    ...mapState('auth', ['registryContractApproved', 'user']),
     ...mapState('web3', ['recordContract']),
 
     done() {
       return this.currentStep === this.numSteps
     },
 
+    formattedBalance() {
+      return formatTokenAmount(this.user.codxBalance)
+    },
+
     currentStep() {
-      if (this.balance.eq(0)) {
+      if (new BigNumber(this.user.codxBalance).eq(0)) {
```

Comment:
I suppose this could just be `this.user.codxBalance === '0'`, but I feel like wrapping it in a BigNumber is safer.